wenet/ssl/bestrq/bestqr_model.py

```diff
@@ -62,93 +174,106 @@ def forward(
         text: Optional[torch.Tensor] = None,
         text_length: Optional[torch.Tensor] = None,
     ):
-        # should support nonstreamming and streamming
-        # TODO(Mddct): streamming future
-        # eg: full attenton and chunk or  dynamic chunk training
-        # 1 forward subsampling
-        xs, pos_emb, masks = self._forward_subsampling(xs, xs_lens)
-        unmasked_xs = xs
-        # 2 mask features
-        # 2.0 apply mask
-        masked_xs, masked_masks = self._apply_mask(xs)
+        # force global cmvn
+        xs = xs - self.signal_mean
+        if self.signal_norm_var:
+            xs = xs * self.signal_istd
+        input = xs
+
+        features_pen: Optional[torch.Tensor] = None
```

Comment:
hi, I guess `xs` here is just the log mel fbank feature from dataloader, so it seems that calculating features_pen is not meaningful in this case.